docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -1612,6 +1612,119 @@ Note/limitations:
 
 - Applying a uniform non-response rate limits the impact of race/ethnicity, age, and sex to affect the sampled population. This might make some aspects of PRL easier as it is less likely the same simulants will be missing from each sample.
 
+Women, Infants, and Children (WIC)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+WIC is a government benefits program designed to support mothers and young 
+children. The main qualifications are income and presence of young children 
+in the home. 
+
+
+**When to Sample** 
+
+- Sample compiled on the time step containing Jan 1st of each year (the time step might end on Jan 2nd, Jan 15th, Jan 27th, etc.)
+
+**What to Sample** 
+
+.. list-table:: Simulant Attribute to Sample 
+  :widths: 20
+  :header-rows: 0
+
+  * - Household number (random indicator which is the same for simulants in the same household)
+  * - Unique simulant ID (for PRL tracking)
+  * - First name
+  * - Middle initial 
+  * - Last name
+  * - Age 
+  * - DOB 
+  * - Home Address 
+  * - Home Zip Code 
+
+Here is an example: 
+
+.. image:: survey_example.png
+
+**Who to Sample** 
+
+Selection for the observer will be in 2 steps: 
+
+#. Eligibilty for WIC 
+#. Covered by WIC 
+
+Please note that multiple people in the same household can be selected and selection 
+should be independent (e.g., having one child in the household on WIC does not mean a 
+second child is or is not on WIC). Inclusion is at the simulant level. Do not include 
```

Comment:
Am I understanding this right that parents/guardians are also selected independently of children? So, you could have a tracked parent sampled here but not the <1 year old child that made them eligible?

Not saying this is an issue, it could just be a limitation, but that would seem to be the more important example to use here instead of/in addition to siblings.

Reply:
Yes, that is my understanding of the program. For example, if a mom is enrolled you get food like bread and cheese, but if an infant is enrolled you get baby formula and stuff. So I think it is independent in WIC, which I continued here. 

I agree, I should have written that more explicitly and added here!